lib/index.js

```diff
@@ -5,10 +5,7 @@ var Resource = require('./Resource'),
     Controllers = require('./Controllers'),
     Errors = require('./Errors'),
     inflection = require('inflection'),
-    _ = require('lodash');
-
-// make deep defaults the default
-_.defaults = require('merge-defaults');
+    mergeDefaults = require('merge-defaults');
```

Comment:
it doesn't seem like you're using _.defaults anywhere (thus needing the default behavior), and the line above this one overrides the default behavior with merge-defaults (so it works that way everywhere)